### PR TITLE
MAT-6431: update for 508 compliance

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-6431](https://jira.cms.gov/browse/MAT-6431)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.scss [hunk 1]
```diff
@@ -45,7 +45,7 @@ $header-weight: 500;
       font-weight: $header-weight;
       padding: 16px;
       text-align: left;
-      border-bottom: 1px solid #cacaca;
+      border-bottom: 1px solid #8c8c8c;
       background-color: #ededed;
```
Comment (adongare) [inline src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.scss]: seems redundant. should inherit from thead?

File: src/components/testCaseLanding/common/TestCase.scss [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
-  border-bottom: solid 1px #ddd;
+  border-bottom: solid 1px #8c8c8c;
 
   > td {
     padding: 10px;
     max-width: 10vw;
     white-space: normal;
     vertical-align: top;
-    border-bottom: solid 1px #ddd;
+    border-bottom: solid 1px #8c8c8c;
```
Comment (adongare) [inline src/components/testCaseLanding/common/TestCase.scss]: same here

**Review by adongare — DISMISSED** [review]

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.scss (hunk 1 above)
Comment (mcmcphillips) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.scss]: modified

File: src/components/testCaseLanding/common/TestCase.scss (hunk 2 above)
Comment (mcmcphillips) [reply src/components/testCaseLanding/common/TestCase.scss]: modified

**Review by adongare — APPROVED** [review]

**Pull request merged**